daemon/src/to_sse_event.rs

```diff
@@ -32,6 +34,35 @@ pub struct Cfd {
     pub state: CfdState,
     pub actions: Vec<CfdAction>,
     pub state_transition_timestamp: u64,
+
+    pub details: CfdDetails,
+}
+
+#[derive(Debug, Clone, Serialize)]
+pub struct CfdDetails {
+    tx_url_list: Vec<TxUrl>,
+    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_btc::opt")]
+    payout: Option<Amount>,
+}
```

Comment:
Is this type necessary? Can we not just have `tx_urls` and `payout` in the `Cfd` type?